src/sources/forus-webshop/pug/tpl/pages/product.pug

```diff
@@ -30,6 +30,7 @@ section.section.section-product: .wrapper
                     .product-price-old(ng-bind="$ctrl.product.old_price | currency_format")
                     .product-price-insufficient(
```

Comment:
@ateamcms could add a tooltip behind .product-price-insufficient what is also only visible with the same ng-show/ng-if restrictions

Reply:
lets add some more context for the user why he can't buy the product with such a tooltip

Reply:
@maxvisser 👍